### DISCUSSION TITLE
Latest EMQX with docker keep crashing

### DISCUSSION THREAD

**Discussion by GaoShin** [emqx/emqx#8594] [General]
### What happened?

Latest EMQX docker container keeps crashing

### What did you expect to happen?

Work as normal

### EMQX version

Cannot get specific version but on docker hub,I can find the image build date:
Jul 3, 2022 at 6:43 pm
And it's DIGEST as below: 
DIGEST: 1b4ba3b45c43

Image SHA256:
sha256:96e74553da07897620cd81144b8f36a1c4432e73f204a5698c2069467c8e079f 

### OS version

Ubuntu 20.04.3 LTS

### Log files

Keep output the following two lines
<details>
ERROR: call_hocon_failed: -s emqx_conf_schema -c /opt/emqx/etc/emqx.conf multi_get node.data_dir node.name node.cookie node.db_backend cluster.proto_dist

[error]: {scan_error,<<"illegal characters \"/c\" line_number 274 in_file /opt/emqx/etc/emqx.conf">>}

</details>

The conf file line 274 is :
node.crash_dump = log/crash.dump

**Comment by ieQu1** [comment]
Hello,

It looks like you are trying to use 4.* config for EMQX 5.0. Their configuration file formats are incompatible; 5.0 uses HOCON format where string values should be wrapped in `"`s.

**Reply by GaoShin** [reply]
Really do upgrade from 4.X to latest.But it should also upgrade the config file

**Reply by ieQu1** [reply]
I see, thanks for the answer. There is no upgrade path from 4.0 to 5.0, because a lot of EMQX subsystems have been completely rewritten, so there is no direct mapping from 4.0 to 5.0 config.
